src/server/pfs/server/server_test.go

```diff
@@ -32,12 +32,14 @@ import (
 	"github.com/pachyderm/pachyderm/src/server/pkg/uuid"
 
 	etcd "github.com/coreos/etcd/clientv3"
+	"github.com/hashicorp/golang-lru"
 	"golang.org/x/net/context"
 	"golang.org/x/sync/errgroup"
 	"google.golang.org/grpc"
 )
 
 const (
+	testingTreeCacheSize       = 8
```

Comment:
If we export the default, seems like we could re-use it here. I don't see the harm in exporting it for this reason. I believe we do this for other constants. It's a bit DRYer

Reply:
This is slightly more complicated than you might think—PFS doesn't define this constant anymore, because PFS doesn't create the tree cache. Instead, `pachd/main.go` defines this constant and creates the tree cache, and since it defines a main function, it can't be imported:
```
$ go test -c ./src/server/pfs/server/
# github.com/pachyderm/pachyderm/src/server/pfs/server
src/server/pfs/server/server_test.go:28:2: import "github.com/pachyderm/pachyderm/src/server/cmd/pachd" is a program, not an importable package
```

Reply:
Per our conversation, I'll leave the PR as-is.